index.php

```diff
@@ -28,7 +30,7 @@
         </div>
         <div class="article-author font-serif">
           <?php echo __('[:es]Por[:en]By'); ?>:<br/>
-          <?php the_author(); ?>
+          <?php echo !empty($author) ? $author : ''; ?>
```

Comment:
i mean, if the author is empty we shouldn't output the whole `.article-author` no?